cmd/finch/nerdctl_remote_test.go

```diff
@@ -132,3 +133,96 @@ func TestNerdctlCommand_withVMErrors(t *testing.T) {
 		})
 	}
 }
+
+func TestNerdctlCommand_withEmptyArgs(t *testing.T) {
```

Comment:
I wonder if we can simplify this test by just ensuring that creating and running the command doesn't result in a panic. It feels like this does a lot to ensure that certain variables are set but all we want to do is check the command output IMO.

Reply:
This is a unit test so I don't think it has to run per se, but I do believe the path it follows to mock command runs should panic if you run it without your changes and pass if you run it with your changes